.github/workflows/cfn-analysis.yml

```diff
@@ -26,21 +26,11 @@ jobs:
           node-version: 14
       - name: Install npm dependencies
         run: yarn install
-      - name: Package main sls into cfn for analysis
+      - name: synthesize cdk template for cfn for analysis
         run: |
-          yarn run serverless-package --stage dev --region us-west-2 --useHapiValidator true --conceal
-      - name: cfn_nag on main serverless
+          yarn cdk synth -c issuerEndpoint='test' -c oAuth2ApiEndpoint='test' -c patientPickerEndpoint='test'
```

Comment:
do we need `--require-approvals=never` here? Not sure if synth also produces a confirmation message if there are security changes or not.